lib/converter.js

```diff
@@ -89,10 +89,14 @@ async function convert (processor, inputFile, options, timings, watch, preview)
   }
   try {
     const page = await browserInstance.goto(`file://${tempFile}`, preview)
+    const puppeteerDefaultTimeout = process.env.PUPPETEER_DEFAULT_TIMEOUT
```

Comment:
could be taken from browser instance but in terms of dependency reduction this might be better